pick.c

```diff
@@ -814,6 +827,10 @@ get_key(char *buf, size_t size, size_t *nread)
 		KEY(LEFT,	"\033[D"),
 		KEY(PAGE_DOWN,	"\033[6~"),
 		KEY(PAGE_UP,	"\033[5~"),
+		KEY(HOME,	"\033[7~"),
```

Comment:
Where did you find the sequences for `HOME` and `END`? Doesn't match the sequences on my machine, this works however:

```c
KEY(END,        "\033[4~"),
KEY(HOME,       "\033[1~"),
```

As mentioned above, keep the list sorted.